platform/ios/src/MGLMapboxEvents.m

```diff
@@ -12,8 +12,8 @@
 
 #include <sys/sysctl.h>
 
-static const NSUInteger version = 1;
```

Comment:
Good question. I thought `version` was used in constructing the API endpoint, [but it is not](https://github.com/mapbox/mapbox-gl-native/blob/e33e58f390b6310d48775554211a64f63b912c54/platform/ios/src/MGLMapboxEvents.m#L587). As it stands, `version` is sent with every event (and that's the only place it is used).

We bumped the user agent version to 1.1 when the last round of changes happened, at @camilleanne's request.